src/components/Pagination/Pagination.js

```diff
@@ -1,34 +1,33 @@
-import { useCallback, useMemo } from "react";
+import { useEffect, useState } from "react";
 import arrow from "../../assets/images/icons/ic_pagination_arrow.svg";
 import styles from "./Pagination.module.scss";
 
 const LIMIT = 5;
 const Pagination = ({ pageSize, totalCount, currentPage, setCurrentPage }) => {
-  // 페이지네이션 5개(LIMIT)로 끊어서 2차원 배열로 생성
-  const calcPager = useMemo(() => {
-    if (!totalCount) return;
-
-    const totalPager = Math.ceil(totalCount / pageSize);
+  const [currentGroup, setCurrentGroup] = useState([]);
+  const [totalPager, setTotalPager] = useState(0);
```

Comment:
```suggestion
  const [totalPage, setTotalPage] = useState(0);
```

뒤에 r이 실수로 붙여져있는것같네요!